### ISSUE TITLE
[Apple Notes] Import selected folders feature

### ISSUE THREAD

**Issue by naga-re** [obsidianmd/obsidian-importer#149]
Currently, you can only import all folders and notes from Apple Notes into Obisdian.

However, for various reasons, you may only want to import certain folders and notes into Obsidian.

Adding a window before the actual import to select the files you wish to import would be a good thing.

An application like [Exporter](http://falcon.star-lord.me/exporter/) offers it. It therefore seems that there is no real technical obstacle to adding this function

<img width="299" alt="Capture d’écran 2023-10-21 à 11 45 46" src="https://github.com/obsidianmd/obsidian-importer/assets/98703770/ad31f554-9c49-45a9-aad7-cc0caf74a88d">

**Comment by mirnovov** [comment]
The Apple Notes importer uses very different technology from the Exporter app but it should still be feasible.

**Comment by viethung0823** [comment]
Is this one on the dev roadmap yet? Or at least allow user to select the desire directory destination settings?

**Comment by hmijail** [comment]
@naga-re I agree that this functionality would be great. And given that this is a one-shot thing, and that the Exporter app is free (thank you for the pointer!), why not just use it? 
Just asking because I wonder if I'm missing some reason to avoid the app.

**Comment by OliveJy** [comment]
It really would be a superb improvement.
The import is perfect, but it's really annoying to have to import all the notes when you just want to retrieve one or two new ones.

**Comment by JhCandiceYao** [comment]
Seconding this! For people who use apple notes for other personal purposes outside of knowledge management, it'd be great to be able to only import from the folders that contain relevant notes.

**Comment by tgrosinger** [comment]
Closing this in favor of #473 where it can be implemented in a reusable way.
